web/src/main/java/com/epam/esm/web/controller/UserController.java

```diff
@@ -83,23 +78,32 @@ public UserController(UserService userService, PurchaseService purchaseService,
      *
      * @return list
      */
+    @PreAuthorize("isAuthenticated()")
```

Comment:
again, according to this, every user can see purchases of another user only by specifying the id of this another user
this private info should be accessible only be the owner of the info or an admin